src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -4923,15 +4923,15 @@ T P
                 // (9,9): error CS8374: Cannot ref-assign 'tOut' to 'F' because 'tOut' has a narrower escape scope than 'F'.
                 //         F = ref tOut; // 2
                 Diagnostic(ErrorCode.ERR_RefAssignNarrower, "F = ref tOut").WithArguments("F", "tOut").WithLocation(9, 9),
-                // (10,17): error CS8331: Cannot assign to variable 'in T' because it is a readonly variable
+                // (10,17): error CS8329: Cannot use variable 'in T' as a ref or out value because it is a readonly variable
```

Comment:
>in T [](http://example.com/codeflow?start=63&length=4)

It looks like this diagnostics can benefit from including parameter names by default, I am referring to the other PR you are working on.

Reply:
Actually, it's not necessary to include them by default in this case, the other PR fixes it either way: https://github.com/dotnet/roslyn/pull/64179/files#diff-9ec539dd7cf09e068cb756dd30a9fc573fe309c285783a1e0ccca731dd9e776dR4090

But I agree that it might be beneficial in other cases, though.